openmc/mgxs/library.py

```diff
@@ -773,9 +773,9 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
             nuclide this will be set to 'macro' regardless.
         xs_ids : str
             Cross section set identifier. Defaults to '1m'.
-        order : Scattering order for this dataset entry.  Default is None,
-            which will force the XSdata object to use whatever the maximum
-            order available.
+        order : Scattering order for this data entry.  Default is None,
+            which will force the XSdata object to use whatever the order of the
```

Comment:
this might read more clearly as simply ..."use the order of the Library object".